HACKING.md

```diff
@@ -1,95 +1,69 @@
 # Snapcraft
 
-## Evaluating pull requests
-
-Oftentimes all you want to do is see if a given pull request solves the issue you were having (you reported that bug, right?). You can always run that pull request from source if you want (see below), but there's an easier way: use `snapcraft-pr`. First of all, you need it installed:
-
-    sudo snap install --classic snapcraft-pr
-
-Every pull request on github has an associated ID, which is used across the project to refer to it in comments, etc. It's clearly displayed on each pull request, but you can also determine it from the URL, for example, the ID of `https://github.com/snapcore/snapcraft/pull/2094` is `2094`.
-
-Once you've determined the pull request ID, you can tell `snapcraft-pr` that you want to use it by running:
-
-    snapcraft-pr.init <pull request id>
-
-Once that has completed, you can now use that version of snapcraft almost exactly like how you would normally use the `snapcraft` command, but instead of using `snapcraft`, use `snapcraft-pr <pull request id>`. For example, if you would normally run `snapcraft cleanbuild`, run `snapcraft-pr <pull request id> cleanbuild`.
-
-If the pull request gets updated after you've already done this, you can update your version by simply running `snapcraft-pr.init <pull request id>` again.
-
-
-## Installing from pip
-
-First install a few dependencies:
-
-    sudo apt install gcc g++ make python3-dev python3-venv libffi-dev libsodium-dev libapt-pkg-dev squashfs-tools patchelf execstack rpm2cpio p7zip-full libyaml-dev libxml2-dev libxslt-dev
+## Setting up a development environment
+
+We want to make sure everyone develops using a consistent base, to ensure that these instructions rely on LXD (use whatever is convenient as long as you do not stray away from an Ubuntu 16.04 LTS base)
+
+First we setup LXD:
+
+    sudo snap install lxd
+    sudo lxd init  # If unsure, pick `dir` as the storage backend.
+    sudo adduser "$USER" lxd
+    newgrp lxd
+    lxc create ubuntu:16.04 snapcraft-dev
+    lxc config set snapcraft-dev raw.idmap "both $UID 1000"
+    lxc config device add snapcraft-dev snapcraft-project disk source="$(PWD)" path=/home/ubuntu/snapcraft
+    lxc start snapcraft-dev
+
+The we install the required dependencies:
+
+    lxc exec snapcraft-dev -- apt update
+    lxc exec snapcraft-dev -- apt install \
+        execstack \
+        g++ \
+        gcc \
+        libapt-pkg-dev \
+        libffi-dev \
+        libsodium-dev \
+        libxml2-dev \
+        libxslt-dev
+        libyaml-dev \
+        make \
+        p7zip-full \
+        patchelf \
+        python3-dev \
+        python3-venv \
```

Comment:
You never actually set up a venv. Either you should do that, or you can get rid of this.